Detectors/EMCAL/reconstruction/src/CaloRawFitterGamma2.cxx

```diff
@@ -87,7 +87,7 @@ CaloFitResults CaloRawFitterGamma2::evaluate(const gsl::span<const Bunch> bunchl
     time = time * constants::EMCAL_TIMESAMPLE;
     time -= mL1Phase;
 
-    return CaloFitResults(maxADC, pedEstimate, mAlgo, amp, time, (int)time, chi2, ndf);
+    return CaloFitResults(maxADC, pedEstimate, 0, amp, time, (int)time, chi2, ndf);
```

Comment:
What is this?

Reply:
The third argument in CaloFitResults takes the fit status. While in the old version we were giving the third argument the fitting algorithm instead of the fitting status which wrong. So now it is fixed, and 0 status means the fit is successful.